packages/assets-controllers/src/MultichainAssetsController/MultichainAssetsController.ts

```diff
@@ -57,6 +56,27 @@ export type AssetMetadataResponse = {
   };
 };
 
+export type MultichainAssetsControllerAccountAssetListUpdatedEvent = {
+  type: `${typeof controllerName}:accountAssetListUpdated`;
+  payload: [
+    {
+      assets: Record<
+        string,
+        {
+          added: (
+            | `${string}:${string}/${string}:${string}`
+            | `${string}:${string}/${string}:${string}/${string}`
+          )[];
+          removed: (
+            | `${string}:${string}/${string}:${string}`
+            | `${string}:${string}/${string}:${string}/${string}`
+          )[];
+        }
+      >;
+    },
+  ];
```

Comment:
Let's re-use this one:

```suggestion
  payload: AccountsControllerAccountAssetListUpdatedEvent['payload'];
```